Makefile

```diff
@@ -69,3 +69,28 @@ install:
 .PHONY: install_test
 install_test: OC_PROJECT=$(OC_TEST_PROJECT)
 install_test: install
+
+.PHONY: watch
+watch: $(call make_help,watch,configure and start the watchers & dev servers)
+watch:
+	watchman watch-project .
+	watchman -j < .watch-schema-sqitch.json
+	watchman -j < .watch-status-schema.json
+	watchman -j < .watch-app-yarn.json
+	watchman -j < .watch-app-relay.json
+	watchman -j < .watch-app-format.json
+	watchman -j < .watch-db.json
+	watchman -j < .watch-server.json
+
+.PHONY: unwatch
+unwatch: $(call make_help,unwatch,shut down the watchers but leave the dev servers running)
+unwatch:
+	watchman watch-del-all
+	watchman shutdown-server
+	if command -v launchctl; then launchctl unload ~/Library/LaunchAgents/com.github.facebook.watchman.plist; fi
+
+# TODO: delete this target
+# ...but maybe it's helpful for debugging watchman itself on a mac?
+.PHONY: watch_log
+watch_log:
+	tail -f /usr/local/var/run/watchman/$(shell whoami)-state/log
```

Comment:
`tail: cannot open '/usr/local/var/run/watchman/matthieu-state/log' for reading: No such file or directory`

Reply:
yes, that's the internal log for watchman itself

Reply:
we shouldn't be writing anything to it...unless we forgot to redirect our streams

Reply:
ideally each watcher is writing to `log/*` in the project dir